sips/sip-026/sip-026-sbtc_peg.md

```diff
@@ -0,0 +1,171 @@
+# SIP-028: sBTC Bootstrap
+
+- **SIP Number:** 028
+- **Title:** A Decentralized and Programmable Asset Backed 1:1 with BTC 
+- **Consideration:** Technical, Economics
+- **Type:** Informational
+- **Status:** Draft
+- **Authors:**
+  - Andre Serrano
+  - Ashton Stephens
+  - Joey Yandle
+  - Mårten Blankfors
+  - Jesus Najera
+  - Jude Nelson
+  - Friedger Müffke
+  - Tycho Onnasch
```

Comment:
```suggestion
SIP Number: 028

Title: sBTC Bootstrap - A Decentralized and Programmable Asset Backed 1:1 with BTC 

Authors:
    Andre Serrano
    Ashton Stephens
    Joey Yandle
    Mårten Blankfors
    Jesus Najera
    Jude Nelson
    Friedger Müffke
    Tycho Onnasch

Consideration: Technical, Economics

Type: Informational

Status: Draft

Discussions-To: <link to forum / workgroup discussions???>
```

Reply:
Should `Type: Operation` ? since this describes standard for sBTC signer node operators? 

another option is `Type: Standard` with `Layer: Application` ?